modules/cluster/src/main/java/io/fluo/cluster/runner/AppRunner.java

```diff
@@ -19,6 +19,10 @@
 import java.util.Iterator;
 import java.util.Map;
 
+import io.fluo.core.impl.Notification;
```

Comment:
Do you have your import order set to the order in https://github.com/fluo-io/fluo/wiki/Contributing ?